### PR TITLE
Complete Docs (for 2.x)

### PR DESCRIPTION
**Pull request by misteroneill**
The docs and examples for this plugin were woefully inadequate - this fixes that.

This is rebased on #31 so that needs merging first.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,46 +21,70 @@ with the class `vjs-playlist` to house the playlist menu:
...
-The PlaylistMenu is a regular [video.js
-Component](https://github.com/videojs/video.js/blob/master/docs/guides/components.md)
-so you may pass in [any
-option](https://github.com/videojs/video.js/blob/master/docs/guides/options.md#component-options)
-that is accepted by Components. In addition, this object may contain
-this specialized property:
+### Default Implementation
 
-#### className
+By default, the plugin will search for _the first element in the DOM with the `vjs-playlist` class.
```
Comment (gkatsev) [inline README.md]: missing ending `_`, I assume `the first` was supposed to be emphasized?

File: example.html [hunk 2]
```diff
@@ -2,57 +2,80 @@
...
+      id="video"
+      class="video-js"
+      height="300"
+      width="600"
+      controls>
+      <source src="http://vjs.zencdn.net/v/oceans.mp4" type="video/mp4">
+      <source src="http://vjs.zencdn.net/v/oceans.webm" type="video/webm">
+    </video>
+
+    <ol class="vjs-playlist">
```
Comment (gkatsev) [inline example.html]: shouldn't this change to a `div`?

File: example-custom-element.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      id="video"
+      class="video-js"
+      height="300"
+      width="600"
+      controls>
+      <source src="http://vjs.zencdn.net/v/oceans.mp4" type="video/mp4">
+      <source src="http://vjs.zencdn.net/v/oceans.webm" type="video/webm">
+    </video>
+
+    <ol id="my-custom-element">
```
Comment (gkatsev) [inline example-custom-element.html]: shouldn't this change to a `div`?

File: example-custom-class.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      id="video"
+      class="video-js"
+      height="300"
+      width="600"
+      controls>
+      <source src="http://vjs.zencdn.net/v/oceans.mp4" type="video/mp4">
+      <source src="http://vjs.zencdn.net/v/oceans.webm" type="video/webm">
+    </video>
+
+    <ol class="my-custom-class">
```
Comment (gkatsev) [inline example-custom-class.html]: shouldn't this change to a `div`?

File: example.html (hunk 2 above)
Comment (misteroneill) [inline example.html]: Not for 2.x. I know... confusing. 😕

File: example-custom-element.html (hunk 3 above)
Comment (misteroneill) [inline example-custom-element.html]: Not for 2.x. I know... confusing. 😕

File: example-custom-class.html (hunk 4 above)
Comment (misteroneill) [inline example-custom-class.html]: Not for 2.x. I know... confusing. 😕

**Pull request merged**